Gemfile

```
@@ -9,6 +9,7 @@ gem "mutex_m"
 gem "bigdecimal"
 
 gem "fastlane"
+gem "cocoapods", "~> 1.16"
```

RuboCop (1.82.1)

[convention] Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `cocoapods` should appear before `fastlane`.

(Bundler/OrderedGems)